<launch>

    <arg name="use_left_arm"                  default="true"/>
    <arg name="use_right_arm"                 default="true"/>

    <param name="manual_motion/use_left_arm_" type="boolean" value="$(arg use_left_arm)"/>
    <param name="manual_motion/use_right_arm" type="boolean" value="$(arg use_right_arm)"/>
    <node pkg="victor_hardware_interface" type="manual_motion.py"
          name="manual_motion" output="screen"/>
    
</launch>
